src/components/Lists/useStyles.js

```diff
@@ -10,6 +10,9 @@ const useStyles = makeStyles(({ typography, palette, breakpoints }) => ({
       justifyContent: "space-between",
     },
   },
+  list: {
+    paddingBottom: typography.pxToRem(48),
```

Comment:
Seems to be 80px from the designs?

<img width="467" alt="Screenshot 2022-01-19 at 23 19 30" src="https://user-images.githubusercontent.com/12892109/150207531-6e134704-4622-4e30-a153-81685a145f0f.png">